### ISSUE TITLE
New Doom shard heal not acting as described

### ISSUE THREAD

**Issue by Hussain-maker** [ValveSoftware/Dota2-Gameplay#11513]
### Ability name

Scorched Earth (Aghanims shard upgrade)

### Description

Description of spell

**Heals Doom and controlled units for 60% of the damage values**

Actual outcome

Heals doom for 39hp a second regardless of number of enemies or damage.

See

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/79055482/1659ff4e-2e29-47c2-a4ac-e6b00263cc25)

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/79055482/a84912ab-41eb-4365-8c4b-2352631cb355)

easy to replicate, stand alone with damage taken and use scorched earth it will heal 

stand alone with damage taken near one enemy or more and same heal 39hp a second is received

I guess someone confused the old scorched earth with the new shard? or missed the damage portion + the base heal?

patch notes from wiki
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/79055482/d22d2618-8b13-48c5-9b8c-0fbcd8f7391b)

patch notes from dota website
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/79055482/5b97eafe-60fa-455d-870b-509c65f7cf48)

### Example Match ID (and possibly Timestamp)

1291449666

**Comment by steflomace** [comment]
This is wrong description or not intended behavior. Scorched Earth heals even if no enemies in area. The value of heal is 60% of base damage, so if you buy something on spell amp heal increased.
